NEXT_CHANGELOG.md

```diff
@@ -133,6 +133,15 @@ When receiving requests with invalid content-type/accept header missmatch (e.g m
 
 By [@Meemaw](https://github.com/Meemaw) in https://github.com/apollographql/router/pull/2370
 
+### Add a metric for existing connections ([issue #2384](https://github.com/apollographql/router/issues/2384))
+
+New metrics are available to track the client connections:
+- `apollo_router_session_count` indicates the number of currently connected clients
+- `apollo_router_active_session_count` indicates the number of in flight GraphQL requests from connected clients.
```

Comment:
Add it into the docs too please

Reply:
added in [c9b3942](https://github.com/apollographql/router/pull/2395/commits/c9b394263f90e7fec1a50808e1a5b373f9c076f0)